examples/ui/ui.rs

```diff
@@ -148,11 +145,6 @@ fn setup(mut commands: Commands, asset_server: Res<AssetServer>) {
                                             .with_style(Style {
                                                 flex_shrink: 0.,
                                                 size: Size::new(Val::Undefined, Val::Px(20.)),
-                                                margin: UiRect {
-                                                    left: Val::Auto,
-                                                    right: Val::Auto,
-                                                    ..default()
-                                                },
```

Comment:
Auto margins are *not* the default (or should not be). For margins, "auto" means "take up available space". The default margin is zero.

Reply:
Oh I couldn't work out what you meant here for a moment.
The default for margins is Px(0.) as you'd expect. 
In both cases, if you look above you can see that I replaced the auto margins with an `AlignItems::Center` on the parent.

Reply:
Ah, gotcha.

Reply:
No wait the default in this PR is still `Val::Undefined` I think, that fix is in the remove `Val::Undefined` PR and hasn't been merged yet. It's getting hard to keep track, really do need these tests.